api-processor/src/main/java/com/synopsys/integration/alert/processor/api/extract/model/project/AbstractBomComponentDetails.java

```diff
@@ -24,6 +24,7 @@
 
     private final LinkableItem license;
     private final String usage;
+    private final ComponentUpgradeGuidance componentUpgradeGuidance;
```

Comment:
Based on the fact that only one implementation uses this field, it should not be in the abstract class, but rather in the implementation.